### DISCUSSION TITLE
It's just a simple apreciation post. I'll close it as complete in a day or two..

### DISCUSSION THREAD

**Discussion by iHarryPotter178** [OpenBangla/OpenBangla-Keyboard#385] [General]
I have always used avro keyboard, on windows, but i always disabled the dictionary feature, because it bring my low end pc to a crawl after writing a few sentences. the same happens with ibus-avro. 
But with OpenBangla, it's so fast, I can now use the dictionary feature or suggestion feature, which works really really great. Maybe its so fast because it's written in rust, or whatever. It's just great. Hope to see a long healthy life of OpenBangla.. 
I don't know if it's possible, **Maybe the app also could be ported to rust.**.Just an idea. I'm not a dev. hence can't help..
Great work brothers.. Keep Up the good work. i'll always be thankful to you guys. May Allah bless you all.

**Comment by mominul** [comment]
@iHarryPotter178 Thanks a lot for the appreciation! Really glad that you have found the performance satisfactory, as we really care about the performance and it was a goal to achieve faster performance.

About porting to Rust, yes I plan to port the GUI portion too, but I am waiting for the GUI libraries to be mature enough.

Thanks! ❤️

**Reply by iHarryPotter178** [reply]
It's great to hear that the gui would also be ported.. For rust gui, Libcosmic, which is what system76 is building their cosmic desktop on, would be a great choice, it will get better support and development, or even iced itself, from which Libcosmic is forked..
Any way, great work and thanks a lot.. Keep it up.. May Allah bless you all.

**Comment by iHarryPotter178** [comment]
As it's a appreciation post.. I'm closing it.. But do take a look at Libcosmic from system76.. It's cross platform.. So will work well.
